src/StatusCodeSender.js

```diff
@@ -23,8 +23,8 @@ class StatusCodeSender {
 							error.error = new Errors.GatewayTimeoutError();
 							break;
 
-					default:
-							error.error = new Errors.DefaultError(error && error.payload && error.payload.errors[0] && error.payload.errors[0].message)
+						default:
+								error.error = new Errors.DefaultError(error && error.payload && error.payload.errors[0] && error.payload.errors[0].message)
```

Comment:
I think more whitespace got added to this line by accident. Can you remove that

- [ ] clean up whitespace

Reply:
im not sure why its showing up super weird here but i lined it up to match all the other cases.
![image](https://github.com/smarty/smartystreets-javascript-sdk/assets/126025799/db359671-c33f-45f3-93ae-16555c638d28)

Reply:
Yes it's just the line right under `default:` that has extra indent. I'll can commit the whitespace change I am talking about if that's easier